.cicd/scripts/srw_ftest.sh

```diff
@@ -53,35 +53,32 @@ pwd
 #     "Run the Workflow Using Stand-Alone Scripts".
 echo "BRANCH=${BRANCH}"
 
-# Set the ACCOUNT to use for this PLATFORM ...
-sed "s|^  ACCOUNT: \"\"|  ACCOUNT: \"${SRW_PROJECT}\"|1" -i ush/config_defaults.yaml
-sed "s|hera|${platform,,}|1" ush/config.community.yaml | sed "s|an_account|${SRW_PROJECT}|1" > ush/config.yaml
-
-# Set directory paths ...
-export EXPTDIR=${workspace}/expt_dirs/test_community
-echo "EXPTDIR=${EXPTDIR}"
-sed "s|^workflow:|workflow:\n  EXPT_BASEDIR: ${workspace}/expt_dirs|1" -i ush/config.yaml
-sed "s|^workflow:|workflow:\n  EXEC_SUBDIR: ${workspace}/install_${SRW_COMPILER}/exec|1" -i ush/config.yaml
-
-# Decrease forecast length since we are running all the steps
-sed "s|^  FCST_LEN_HRS: 12|  FCST_LEN_HRS: 6|g" -i ush/config.yaml
-
-# Update compiler 
-sed "s|^  COMPILER: intel|  COMPILER: ${SRW_COMPILER}|g" -i ush/config.yaml
+# Set test settings for the platform and account
+EXPTDIR=${workspace}/expt_dirs/test_community
```

Comment:
`EXPTDIR` is required in the wrapper scripts, so please add export back into this variable:

```suggestion
export EXPTDIR=${workspace}/expt_dirs/test_community
```